rskj-core/src/test/java/co/rsk/peg/federation/FederationStorageProviderImplTests.java

```diff
@@ -924,6 +926,93 @@ void saveFederationElection_shouldSaveInStorage() {
         assertArrayEquals(federationElectionSerializedWithVote, actualAbiCallElectionSerialized);
     }
 
+    @Test
+    void saveFundTransactionUnsignedHash_preLovell700_shouldSaveInStorage() {
```

Comment:
pre lovell 700 it should **not** save in storage right?

Reply:
Updated.